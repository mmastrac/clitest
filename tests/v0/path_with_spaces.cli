#!/usr/bin/env clitest --version=0

$ echo "Path: /AAAAA/BBBBB/CCCCC/DDDDDDDDDDDDDD EEEEEEEEEEEEEEEEEEEEEEEE/"
%EXPECT_FAILURE
! Path: %{PATH}
