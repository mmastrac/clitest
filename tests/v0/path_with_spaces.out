Running <root>/path_with_spaces.cli ...

printf "Path: /AAAAA/BBBBB/CCCCC/DDDDDDDDDDDDDD EEEEEEEEEEEEEEEEEEEEEEEE/\n"
---
Path: /AAAAA/BBBBB/CCCCC/DDDDDDDDDDDDDD EEEEEEEEEEEEEEEEEEEEEEEE/
---
âœ… OK

<root>/path_with_spaces.cli PASSED
1 test(s) passed
